deploy/ocm-agent-operator-managednotifications/10-managednotifications-cr.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
   namespace: openshift-ocm-agent-operator
 spec:
   notifications:
+    - activeBody: |-
+        Your cluster requires you to take action as there are multiple ingress controllers detected. Red Hat SRE strongly recommends using Openshift-managed Custom Domains instead, as multiple ingress controllers can lead to problems with internal and external networking.
```

Comment:
```suggestion
        Your cluster requires you to take action as there are multiple ingress controllers detected. Red Hat SRE strongly recommends using Openshift-managed Custom Domains instead, as multiple ingress controllers, if improperly configured, can lead to problems with internal and external networking.
```

Reply:
Hi @wgordon17 - thank you for the review. I'll amend the message with your suggested changes now.